tests/CMakeLists.txt

```diff
@@ -8,32 +8,30 @@
 # Modified by: Michael E. Tryby
 #              US EPA ORD/NRMRL
 # 
+# Modified by: Velitchko G. Tzatchkov
```

Comment:
Since all changes to this file are cosmetics, I propose to keep it in its original form.

Reply:
Done.